src/components/BookCard.jsx

```diff
@@ -1,6 +1,25 @@
 import PropTypes from 'prop-types';
 import '../styles/BookCard.scss';
 
+function BookList() {
+  const booksDescription = [
+    { title: 'The Hunger Games', author: 'Suzanne Collins' },
+    { title: 'To Kill a Mockingbird', author: 'Harper Lee' },
+    { title: 'The Catcher in the Rye', author: 'J.D. Salinger' },
+  ];
+
+  return (
+    <div className="book-list">
+      {/* {booksDescription.map((book) => (
+        <Book key={book.title} title={book.title} author={book.author} />
+      ))} */}
```

Comment:
[SUGGESTION and OPTIONAL for now] Your project now is approved, but kindly remove the commented code and remember to do this in your next ones, so you can maintain a cleaner project 💯